saray/weather-card/stylesheet.css

```diff
@@ -0,0 +1,45 @@
+.weather-card--container {
+    background-color: #FBED78;
+    border-radius: 4px;
+    width: 300px;
+    padding: 15px 20px;
+    font-family: 'Montserrat', sans-serif;
+    line-height: 1.1em;
+}
+
+.weather-card--city {
+    color: #424242;
+}
+
+.weather-card--city-name {
+    font-size: 25px;
+    color: #424242;
+    font-weight: 300;    
+}
+
+.weather-card--day-night {
+    font-weight: lighter;
+    font-size: 15px;
+}
+
+.weather-card--temperature {
+    font-size: 73px;  
+    position: relative;
```

Comment:
Why are you using position: relative?

Reply:
I have no idea. I copied it from the css styles on dev tools Removed it from my CSS already and looks the same. :)